hack/docs/internal/cronjob-tutorial/writing_tests_controller.go

```diff
@@ -194,6 +191,9 @@ var _ = Describe("CronJob controller", func() {
 			controllerRef := metav1.NewControllerRef(createdCronjob, gvk)
 			testJob.SetOwnerReferences([]metav1.OwnerReference{*controllerRef})
 			Expect(k8sClient.Create(ctx, testJob)).Should(Succeed())
+			// Note that you can't set the status while creating the resource. It needs to be updated after
```

Comment:
Do we need the comment? 
I think we can leave it without the comment right

Reply:
IHMO just remove the comment and then we can get this one merged

Reply:
I wouldn't have this comment in production code, but figured it might be worth it for a tutorial example. I can remove it though

Reply:
I think if we add a comment we need to refine that with something like 

```
// Note that you should not manage the status values while creating the resource. 
// The status field is managed separately to reflect the current state of the resource. 
// Therefore, it should be updated using a PATCH or PUT operation after the resource has been created.
// Additionally, it is recommended to use StatusConditions to manage the status. For further information see: 
// https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
```

Reply:
Pushed the new comment!